streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/serialization/CrossLangSerializer.java

```diff
@@ -38,14 +38,16 @@ public Record deserialize(byte[] bytes) {
     List list = (List) msgPackSerializer.deserialize(bytes);
     Byte typeId = (Byte) list.get(0);
     switch (typeId) {
-      case RECORD_TYPE_ID: {
+      case RECORD_TYPE_ID:
+        {
```

Comment:
the code style here is fomated by spotless?

Reply:
> the code style here is fomated by spotless?

autoformated by spotless plugin.